src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -621,12 +622,12 @@ public void onTraversalStart() {
 
         if (doIncrementalImport) {
             logger.info("Callset Map JSON file will be re-written to " + callsetMapJSONFile);
-            logger.info("Incrementally importing to array - " + workspaceDir + "/" + GenomicsDBConstants.DEFAULT_ARRAY_NAME);
+            logger.info("Incrementally importing to workspace - " + workspaceDir);
```

Comment:
Cleaned up code as the names of the GenomicsDB arrays are no longer constant and are based on the interval.